.travis/push-docs.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+# exit early if any command fails
+set -e
+
+git clone --recursive https://github.com/neomutt/neomutt.github.io
+
+(
+cd neomutt.github.io/code
+
+# Remove the old docs
+git rm -r ./*
```

Comment:
As that makes the travis log currently hard to read, you might want to silence the output of this line.